Add Instruction shortcode. (#984)
GoogleChrome/web.dev: src/site/content/en/fast/minify-css/index.md

> Now you'll inspect the content of the CSS:

Reviewer: ```suggestion
Now inspect the content of the CSS:
```
Instructions should be in imperative mood.

> 1. Open the [Fav Kitties site](https://fav-kitties-animated.glitch.me/) in Chrome (it might take a while for Glitch servers to respond the first time).

Reviewer: ```suggestion
1. Open the [Fav Kitties site](https://fav-kitties-animated.glitch.me/) in Chrome. (It might take a while for Glitch servers to respond the first time.)
```

{% Instruction 'devtools-network', 'ol' %}
1. Click the **CSS** filter.

> 1. Make sure **Disable Cache** is checked and reload the page.

Reviewer: ```suggestion
1. Select the **Disable cache** checkbox.
1. Reload the page.
```

## Verify

{% Instruction 'preview' %}

If you got lost in any previous step, you can click
[here](https://fav-kitties-animated-min.glitch.me/), to open an optimized
version of the site.

To inspect the size and content of the files:

{% Instruction 'devtools-network', 'ol' %}
1. Click the **CSS** filter.

> 1. Make sure **Disable Cache** is checked and reload the page.

Reviewer: ```suggestion
1. Select the **Disable cache** checkbox if it isn't already.
1. Reload the page.
```
